pos_plate_number/models/pos_config.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2023 KMEE (http://www.kmee.com.br)
+# @author: Felipe Zago Rodrigues <felipe.zago@kmee.com.br>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    use_plate_number = fields.Boolean(string="Use Plate Number")
+
+    plate_number_generation = fields.Selection(
```

Comment:
Is there a specific reason for having what type of plate number generation was or is it just to save the info?

Reply:
Its just an option, to ask for the plate number or generate id based on the order uid.